client/src/utils/overrides.css

```diff
@@ -0,0 +1,7 @@
+.ant-input-number-group-addon {
+    padding: 0 !important;
+}
+
+.ant-input-number-input {
+    text-align: center !important;
+}
```

Comment:
These overrides makes the quantity selector look good indeed, but it makes all the other number inputs look quite wonky...

Reply:
I was able to do the text-align using CSS ids (unfortunately it doesn't seem to allow !important on inline styles). I had to switch away from using the input addons because they don't accept ids. It doesn't look exactly as nice as it did in this commit but it does look similar and avoids messing other inputs.